ansible/roles/rbenv/tasks/main.yml

```diff
@@ -51,13 +51,6 @@
   become: true
   become_user: deploy
 
-- name: install ruby 2.5.1
```

Comment:
This is slightly unrelated, we don't need to install ruby 2.5.1 anymore. All our envs have moved to 2.6.6.

Reply:
Ha, nice.